sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/Models/TargetIntentResult.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using Azure.Core;
+
+namespace Azure.AI.Language.Conversations
+{
+    /// <summary> This is the base class of an intent prediction. </summary>
+    public partial class TargetIntentResult
+    {
+        /// <summary> This discriminator property specifies the type of the target project that returns the response. </summary>
+        [CodeGenMember("TargetKind")]
+        public TargetKind TargetKind { get; set; }
```

Comment:
It's the same name. Was this necessary?

Reply:
Not worth holding up the PR, though. We can fix it later if unnecessary.

Reply:
Yea I double checked now and it doesn't seem like it was necessary. Will remove it in the next PR.